src/main/java/duke/exception/DukeDateTimeException.java

```diff
@@ -0,0 +1,8 @@
+package duke.exception;
+
+public class DukeDateTimeException extends DukeException {
+    
+    public DukeDateTimeException() {
+        super("Please input a date in the following format: dd/MM/yyyy HH:mm");
+    }

```

Comment:
I like that there are different type of duke exceptions as it helps with abstraction and removes the need to copy the error message for that specific exception.